File: CS/WindowsApplication1/MyVGridControl.cs
using System;
using DevExpress.XtraVerticalGrid;
using DevExpress.XtraVerticalGrid.Painters;

namespace WindowsApplication1
{
    public class MyVGridControl : VGridControl
    {

        public MyVGridControl()
        {

        }

        protected override VGridPainter CreatePainterCore(PaintEventHelper eventHelper)
        {
            return new MyVGridPainter(eventHelper);
        }
    }
}

File: VB/WindowsApplication1/MyVGridControl.vb
Imports System
Imports DevExpress.XtraVerticalGrid
Imports DevExpress.XtraVerticalGrid.Painters

Namespace WindowsApplication1
	Public Class MyVGridControl
		Inherits VGridControl

		Public Sub New()

		End Sub

		Protected Overrides Function CreatePainterCore(ByVal eventHelper As PaintEventHelper) As VGridPainter
			Return New MyVGridPainter(eventHelper)
		End Function
	End Class
End Namespace
